server.js

```diff
@@ -105,6 +105,22 @@ app.get('/robots.txt', (req, res)=>{
 	return res.sendFile(`${__dirname}/robots.txt`);
 });
 
+//Home page
+app.get('/', async (req, res, next)=>{
+	req.welcomeText = welcomeText;
+	return next();
```

Comment:
Can we make a minimal brew here too?

```suggestion
	const brew = {
		title : 'WelcomeText',
		text  : welcomeText
	};
	req.brew = brew;
	return next();
```

And then in Homebrew.jsx have `welcomeText={this.props.brew.text}`

Reply:
This is slightly different because HomePage compares welcomeText to brew.text to detect if any changes have been made and thus Save should be available. But we should be able to do that via a brew.

Reply:
I have now pushed `welcomeText` into the `brew` object on the HomePage, and changed the check from `this.props.welcomeText` to `this.props.brew.welcomeText`.